### PR TITLE
Shrink down AccessibleObject usage

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

File: tests/Linter/ProcessLinterProcessBuilderTest.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+     *
+     * @testWith ["php", "foo.php", "php -l foo.php"]
+     *           ["C:\\Program Files\\php\\php.exe", "foo bar\\baz.php", "\"C:\\Program Files\\php\\php.exe\" -l \"foo bar\\baz.php\""]
+     * @requires OS ^Win
+     */
+    public function testPrepareCommandOnPhpOnWindows($executable, $file, $expected)
+    {
+        $this->assertSame(
+            $expected,
+            (new ProcessLinterProcessBuilder($executable))->build($file)->getCommandLine()
```
Comment (keradus) [inline tests/Linter/ProcessLinterProcessBuilderTest.php]: 5.3 syntax

File: src/Linter/ProcessLinterProcessBuilder.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    /**
+     * @param string $path
+     *
+     * @return Process
+     */
+    public function build($path)
+    {
+        // @TODO change at 2.4 to:
```
Comment (GrahamCampbell) [inline src/Linter/ProcessLinterProcessBuilder.php]: Annotations aren't valid in `//` style comments I don't think?

File: src/Linter/ProcessLinterProcessBuilder.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+ */
+final class ProcessLinterProcessBuilder
+{
+    /**
+     * @var string
+     */
+    private $executable;
+
+    /**
+     * @param string $executable PHP executable, null for autodetection
```
Comment (julienfalque) [inline src/Linter/ProcessLinterProcessBuilder.php]: You removed `null` from original allowed types but forgot to update the description.

File: src/Linter/ProcessLinterProcessBuilder.php (hunk 2 above)
Comment (julienfalque) [inline src/Linter/ProcessLinterProcessBuilder.php]: The point is to have those listed by IDEs (which are pretty lenient with this) and ultimately removed, so I'd say this is perfectly fine.

Comment (keradus) [inline src/Linter/ProcessLinterProcessBuilder.php]: this is not an annotation. it's fine.

File: src/Linter/ProcessLinterProcessBuilder.php (hunk 3 above)
Comment (keradus) [inline src/Linter/ProcessLinterProcessBuilder.php]: nice catch

**Pull request closed without merging**